src/Tests/Microsoft.NET.Build.Tests/GivenFrameworkReferences.cs

```diff
@@ -640,6 +685,78 @@ private List<string> GetRuntimeFrameworks(string runtimeConfigPath)
             return resolvedVersions;
         }
 
+        private Dictionary<string, List<(string asset, string isTrimmable)>> GetRuntimeAssetTrimInfo(TestProject testProject,
+            Action<XDocument> projectChanges = null,
+            [CallerMemberName] string callingMethod = null,
+            string identifier = null)
+        {
+            testProject.Name = "TrimInfoTest";
+            testProject.TargetFrameworks = "netcoreapp3.0";
+            testProject.IsSdkProject = true;
+            testProject.IsExe = true;
+            testProject.RuntimeIdentifier = EnvironmentInfo.GetCompatibleRid(testProject.TargetFrameworks);
+
+            var testAsset = _testAssetsManager.CreateTestProject(testProject, callingMethod, identifier)
+                .WithProjectChanges(project =>
+                {
+                    string writeRuntimeAssetTrimInfoTarget = @"
+<Target Name=`WriteRuntimeAssetTrimInfo` DependsOnTargets=`ComputeFilesToPublish`>
+    <ItemGroup>
+        <LinesToWrite Include=`%(ResolvedFileToPublish.Identity)%09%(ResolvedFileToPublish.PackageName)%09%(ResolvedFileToPublish.IsTrimmable)` />
+    </ItemGroup>
+    <WriteLinesToFile File=`$(OutputPath)runtimeassettriminfo.txt`
```

Comment:
You should be able to use the existing GetValuesCommand for this.

Reply:
Oh that looks much better, thanks!

Reply:
I've addressed your feedback. Would you mind taking another look?